src/Features/Vendor/BalanceSheet/mutations.js

```diff
@@ -37,10 +37,10 @@ export const DeleteBalanceById = () => {
     },
 
     {
-      onSuccess: (data, variables, context) => {
+      onSuccess: () => {
         // success!
       },
-      onError: (error, variables, context) => {
+      onError: () => {
```

Comment:
When there is not any action that needs to be performed on success and error just remove both !!